internal/store/deployment_test.go

```diff
@@ -104,7 +106,8 @@ func TestDeploymentStore(t *testing.T) {
 				},
 			},
 			Want: metadata + `
-        kube_deployment_created{deployment="depl1",namespace="ns1"} 1.5e+09
+		kube_deployment_created{deployment="depl1",namespace="ns1"} 1.5e+09
```

Comment:
Can you please clean up the formating of this and below as well, thanks!

Reply:
Done. 👍